api/docs/release.dox

```diff
@@ -136,6 +136,16 @@ changes:
    calls changed to contain the actual return value, rather than just whether
    successful.  A new marker #dynamorio::drmemtrace::TRACE_MARKER_TYPE_SYSCALL_FAILED
    was added to indicate failure.
+ - Fixed a bug in the AArch64 codec with the way that SVE scalar+immediate predicated
```

Comment:
You've put this under "compatibility changes" but as a bug fix, even though it's changing behavior, I don't think we'd consider it a compatibility breakage.  I would suggest moving it to the start (to make it visible) of the list just below, line 151.

Reply:
Done.